challenge/src/components/SomeInformationForTimeTravelers.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+export default class SomeInformationForTimeTravelers extends React.Component {
```

Comment:
hilariously js is stupid and its you do this

Reply:
hauh i dont get it

Reply:
the component is still named wrong in this file, but it compiles. fun excersize not required for this pr, but find out why and ping me.
please also fix the name here

Reply:
excersize